rkt/pubkey/pubkey.go

```diff
@@ -234,22 +236,32 @@ func downloadKey(u *url.URL, skipTLSCheck bool) (*os.File, error) {
 	return retTf, nil
 }
 
-func getClient(skipTLSCheck bool) *http.Client {
-	if !skipTLSCheck {
-		return http.DefaultClient
+func newClient(skipTLSCheck bool) *http.Client {
+	dialer := &net.Dialer{
+		Timeout:   30 * time.Second,
+		KeepAlive: 30 * time.Second,
+	} // values taken from stdlib v1.5.3
+
+	tr := &http.Transport{
```

Comment:
When I was looking at the problem I found the following paragraph in the `http.Transport` documentation:

```
Transports should be reused instead of created as needed. Transports are safe for concurrent use by multiple goroutines.
```

Not sure if this is really applicable in this case as this part of the code is not running for long.